### PR TITLE
Add a process for generating a D&I report

### PR DESCRIPTION
**Pull request by GeorgLink**
This pull request will add a file "process" to the repository. The process are general steps that any community can take to produce a D&I report. 

Daniel, Dawn, and I came up with the process for the D&I report tutorial at OSSEU but think it will be of use to the D&I workgroup.

Editable google doc: https://docs.google.com/document/d/1T_WZIj8gH6EoS3szL1NbGBX70R95EmhIbAP_KyMA2qY/edit

Please comment here on the pull request or in the google doc.

### CODE REVIEW

**Comment by emmairwin** [comment]
I'm sorry I missed the past calls, but given we don't have all of the goal-metrics, questions, and potential answers identified (with research backing), and standards for privacy/ethics how can we imagine we can create a standard for running a report?

**Comment by GeorgLink** [comment]
Thanks @emmairwin !

Yes. The process will be most useful after we have the resource pages. The main reason for writing down a process is the OSSEU tutorial. We thought it might be helpful to talk through steps involved in creating a diversity and inclusion report.

I pull in aquestion posted on the google doc:
> Doesn;'t this describe our workflow for all of the D&I data/metrics work?
> I'm sorry I haven't been on the call, but this seems like we are jumping ahead ?

Yes. The idea of the process is to be generic and apply regardless of the specific metrics. Part of the process is to consider goals of the community and choose metrics to reach those goales. 

I consider this process an early draft to keep in mind as we continue working on the resource pages.

**Comment by emmairwin** [comment]
So, I am sorry to be difficult on this one, but I think there are goals for a presentation and in generating an actual resource for guidance as part of this repo.

Something like this, I believe should live in a folder that identifies it as being a brainstorm, a draft, a presentation, but not as a source for running a D&I report.  For these reasons, that are already deeply on my mind:

1) CHAOSS should have strong opinions, and guidance for how this work should (and should not) be used. I think using words like  'anyone community run a d&i report is a huge flag that we're not ready;.  Related article - what it means to be an Engineer :  https://www.nspe.org/resources/ethics/code-ethics 
2) We should have guidance for how goals are set, and not just suggest that people have goals.
3) Demographic data - legal, compliance (including GDPR) and understanding risk to people. 

This is a broader conversation perhaps about who WE want to be in proving this information, are we just collecting methods?   Are we holding ourselves, and others accountable for how this work is being used, and perhaps misused?  Rather than say 'Any community can do this', can we say  ' Any community can do this, but only communities with x CHAOSS badge are verified to do so.   Kind of like https://en.wikipedia.org/wiki/Iron_Ring.     

To summarize, I want to propose this just stay a talk (and live elsehwere), and not be a proposed resource yet. I don't think we're there.

**Comment by GeorgLink** [comment]
Thanks Emma,

I think you make good points and I agree that we have more ground to cover.
I would be happy to keep this in limbo as a talking point (maybe issue)
rather than move it into our repository quite yet.

Georg

**Comment by emmairwin** [comment]
I'll lose this for now, and you can open a new PR as per recommendations in the Meeting yesterday.  Thanks!
